cmd/kube-scheduler/app/server.go

```diff
@@ -278,7 +278,6 @@ func buildHandlerChain(handler http.Handler, authn authenticator.Request, authz
 	handler = genericapifilters.WithRequestInfo(handler, requestInfoResolver)
 	handler = genericapifilters.WithAuthorization(handler, authz, legacyscheme.Codecs)
 	handler = genericapifilters.WithAuthentication(handler, authn, failedHandler, nil)
-	handler = genericapifilters.WithRequestInfo(handler, requestInfoResolver)
```

Comment:
all other handler chains wrap WithRequestInfo last, so that other filters (authn/authz) have access to request info if needed. prefer keeping the second call and dropping the first